### PR TITLE
update-oddcontrast

### PR DESCRIPTION
**Pull request by SondraE**
## Steps to test/reproduce
- OddContrast tool page
- the three images are updated to show the current state of OddContrast

### CODE REVIEW

**Review by jamesnw — APPROVED** [review]
Looks good to me! I deleted the `contrast.jpg` image, because I don't think it was needed. Let me know if it is needed, and I should restore it.

File: README.md [hunk 1]
```diff
@@ -22,8 +22,9 @@ if necessary, then run `nvm install` (once per active shell) to use the correct
 version of Node for OddSite development.
 
 The correct [Yarn](https://yarnpkg.com/) version is included with Node, and will
-be used automatically for any `yarn` command. To activate it, run `corepack
-enable` (once per local Node installation).
+be used automatically for any `yarn` command. To activate it, run
+`corepack enable`
+(once per local Node installation).
```
Comment (jgerigmeyer) [inline README.md:27]: Out of curiosity, why this change?

**Review by jgerigmeyer — APPROVED** [review]
